workflows/scripts/normalize.py

```diff
@@ -40,36 +40,27 @@ def normalization(
 
     # loading parameters
     logging.info(f"Loading Annotation configuration from: {config}")
+    normalize_config = config["params"]
 
-    normalize_obj = Path(config)
-    normalize_config_path = normalize_obj.absolute()
-    if not normalize_obj.is_file():
-        e_msg = "Unable to find Normalization configuration file"
-        logging.error(e_msg)
-        raise FileNotFoundError(e_msg)
-
-    with open(normalize_config_path, "r") as yaml_contents:
-        normalize_config = yaml.safe_load(yaml_contents)["normalize_configs"]["params"]
-        logging.info("Annotation configuration loaded")
-
-    meta_features = [
-        "Metadata_Plate",
-        "Metadata_Well",
-        "Metadata_WellRow",
-        "Metadata_WellCol",
-        "Metadata_gene_name",
-        "Metadata_pert_name",
-        "Metadata_broad_sample",
-        "Metadata_cell_line",
-    ]
+    # -- this is form old version of CellProfiler
+    # meta_features = [
+    #     "Metadata_Plate",
+    #     "Metadata_Well",
+    #     "Metadata_WellRow",
+    #     "Metadata_WellCol",
+    #     "Metadata_gene_name",
+    #     "Metadata_pert_name",
+    #     "Metadata_broad_sample",
+    #     "Metadata_cell_line",
+    # ]
 
     # normalizing annotated aggregated profiles
     logging.info(f"Normalizing annotated aggregated profiles: {anno_file}")
     normalize(
         anno_file,
         features=normalize_config["features"],
         image_features=normalize_config["image_features"],
-        meta_features=meta_features,
+        meta_features=normalize_config["meta_features"],
```

Comment:
Are meta_features specified in the config file?

Is there no way to have it automatically find columns in a file starting with Metadata (which CellProfiler consistently names all metadata columns this way) instead of a user having to name all of the metadata columns?

Also, the name `meta_features` is a bit confusing to me. From what I can see from the commented-out code, it is just the metadata column names so I would recommend to make the name clearer.

Reply:
Yes meta_features is defined in the workflow config file. To answer your question about the name, we are keeping the name consistent with the function parameters. This will make it easier for users to search it up in the documentation. 

Here's the [documentation](https://github.com/cytomining/pycytominer/blob/a5ae6c81a275b692ef5d4c85cfeb37696bf69242/pycytominer/normalize.py#L44-L47) if you're curious. 

It seems that `pycytominer` get's it's meta features from column names. I do not know if it can do what you're asking. Maybe @gwaybio may have a better answer. In addition, you can overwrite the `infer` default parameter in `meta_features` and provide your own list of features in the config.

Reply:
pycytominer identifies `meta_features` using `Metadata_` column name prefixes if and only if `meta_features='infer'`. 

We will almost always use `meta_features='infer'`, but we want flexibility to define metadata columns if necessary